### DISCUSSION TITLE
Drag placeholder for invalid drag targets

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#4992] [Ideas]
It would be nice to at least have the option to show a red drag placeholder for when you are trying to drop a block somewhere it can not be dropped; so that it is more clear for the end user that it can not be dropped. 

It would be event better to be able to show a message while drag-hovering invalid drag targets. (for example: "Input blocks can only be dropped inside a Form block"). If not while drag-hovering, at least fire an event when dropped like editor.on('block:drag:stop:invalid', fn) or similar.

Just to be clear, I am talking about the 'draggable' property of Components that allow a block to only be dropped inside certain elements.

        draggable: 'form, form *', // Can be dropped only inside `form` element

**Comment by artf** [comment]
Did you try to check the `sorter:drag:end` event, it should contain some info in regards
